presto-main/src/test/java/com/facebook/presto/metadata/AbstractMockMetadata.java

```diff
@@ -114,6 +115,12 @@ public boolean schemaExists(Session session, CatalogSchemaName schema)
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    public Map<ColumnHandle, ColumnHandle> pushdownSubfieldPruning(Session session, TableHandle tableHandle, Map<ColumnHandle, List<SubfieldPath>> desiredSubfields)
+    {
+        throw new UnsupportedOperationException();
```

Comment:
Any harm in returning empty map?

Reply:
@tdcmeehan All other methods here throw, hence, I did the same.